src/k8s_client/k8s_client.go

```diff
@@ -63,6 +64,10 @@ type K8SClient interface {
 	GetPods(namespace string, labelMatch map[string]string, fieldSelector string) ([]v1.Pod, error)
 	GetCSV(namespace string, name string) (*olmv1alpha1.ClusterServiceVersion, error)
 	GetCSVFromSubscription(namespace string, name string) (string, error)
+	GetSubscription(subscription types.NamespacedName) (*olmv1alpha1.Subscription, error)
+	GetAllInstallPlansOfSubscription(subscription types.NamespacedName) ([]olmv1alpha1.InstallPlan, error)
+	MakeSubscriptionAutomatic(subscription types.NamespacedName) error
```

Comment:
Don't forget to also remove the interface / re-generate mocks

Reply:
missed it